unit/src/MooseServerTest.C

```diff
@@ -1571,51 +1573,72 @@ TEST_F(MooseServerTest, DocumentReferencesRequest)
 
 TEST_F(MooseServerTest, DocumentFormattingRequest)
 {
+  // put variables file on disk which will be included from base input file
+
+  std::ofstream include_variables("include_variables.i");
+  include_variables << "[Variables]\n  [u]\n  []\n[]\n";
+  include_variables.close();
+
   // didchange test parameters - update input to set up document formatting
 
   std::string doc_uri = wasp::lsp::m_uri_prefix + std::string("/test/input/path");
   int doc_version = 4;
   std::string doc_text_change = R"INPUT(
 
-[./Mesh] type = GeneratedMesh
-       dim =   2         patch_size  =    40
-patch_update_strategy = AUTO
+[Mesh]
+type=GeneratedMesh   dim   =  1
+    []
 
-        displacements = 'disp_x disp_y'
-     parallel_type = REPLICATED   bias_x = 1
+[Functions]
+[./multi_line_indent_increase_01]
+type = PiecewiseConstant
+xy_data = "0 0.0
+           1 1.0
+           10 3.0
+           100 2.0
+           1000 4.0"
+[../]
+[./multi_line_indent_increase_02]
+type = PiecewiseConstant
+xy_data = " 0 0.0
+            1 11.0
+           10 33.0
+          100 22.30
+         1000 42.210"
+[../]
+[./multi_line_indent_decrease_01]
+            type = PiecewiseConstant
+            xy_data = "100 1.0
+                       200 12.0
+                       300 313.0
+                       400 4514.0
+                       500 45615.0"
+[../]
+[./multi_line_indent_decrease_02]
+            type = PiecewiseConstant
+            xy_data = "100 1.0
+  200 12.0
+                       500 45615.0"
 [../]
 
-[./Variables] [u]
-  [../]      [./v] [] [abc]
-
-
-    order     = CONSTANT
-    type = MooseVariableConstMonomial
-  []   []
-[AuxVariables]  [aux_name]
-type = MooseVariableBase family =   LAGRANGE
-
-                []       [] [./AuxVariables]
-  [disp_x]   [] [./xyz]
-type  = MooseVariable   family  =   MONOMIAL
-      []
- [disp_y] []    [disp_z]
-    type = MooseVariableBase    [../]
-[../]      [BCs]
-
-    [./all]  type = VacuumBC
-  boundary   =   'left right' variable = abc
-
-   displacements  =  'xyz abc xyz disp_x' []
-[../] [./Executioner]
-type = Transient
-       [../]    [Problem] solve = false   []
```

Comment:
why does this test change so much?

Reply:
Because I completely changed the starting input that gets formatted to test the actual formatting features.

Instead of the old input that just contained arbitrary spacing, I updated the input so now it tests things like:

 - multiline string values get relative indentation together
 - long double quoted strings being reflowed across lines
 - long single quoted strings do not get broken over lines
 - blank lines are preserved and not collapsed like before
 - inline comments on fields, `!include`, section headers
 - the `!include` directive to ensure it no longer expands

I based the reflow and relative indentation of double quoted vs single quoted strings on your [reflow.i](https://github.com/idaholab/moose/blob/next/framework/contrib/hit/test/reflow.i) input.

The previous input did not test any of these things, and subsequently many of those features did not work.